scalableminds/webknossos
# allow team managers to activate users

### URL of deployed dev instance (used for testing):
- https://teammanagercanactivate.webknossos.xyz

### Steps to test:
- create user, activate as scmboy, give him team manager rights
- register third user, log in as the team manager, try to activate the third user
- should work

### Issues:
- fixes #3043 

------
